frontend/dockerfile/docs/reference.md

```diff
@@ -59,8 +59,8 @@ Line continuation characters are not supported in comments.
 > equivalent:
 >
 > ```dockerfile
->         # this is a comment-line
->     RUN echo hello
+> # this is a comment-line
+> RUN echo hello
```

Comment:
Thanks for the PR :tada: Unfortunately the spacing is actually deliberate.

See the comment above:

> For backward compatibility, leading whitespace before comments (`#`) and
> instructions (such as `RUN`) are ignored, but discouraged. Leading whitespace
> is not preserved in these cases, and the following examples are therefore